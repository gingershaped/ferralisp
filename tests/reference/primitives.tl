() 42 (q(1 2 3))
(q Here's;a:valid-name.]even[`though"it~has*special$chars&and#numb3rs!!)
( q (q  x) )