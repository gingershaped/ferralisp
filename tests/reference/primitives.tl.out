()
42
(1 2 3)
Here's;a:valid-name.]even[`though"it~has*special$chars&and#numb3rs!!
(q x)